index.html

```diff
@@ -1051,16 +1051,28 @@ <h3>Data delivery</h3>
               and {{PressureRecord/[[Time]]}} set to |timestamp|.
             </li>
             <li>
-              If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
-              set observer.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] to |record|
-              and [=iteration/continue=].
+              If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] [=map/exists]:
+              <ol>
+                <li>
+                  Set |observer|.{{PressureObserver/[[LastRecordMap]]}}[|source|] to |record|.
+                </li>
+                <li>
+                  Set |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|] to |record| and [=iteration/continue=].
```

Comment:
maybe nicer with the continue as a separate step

Reply:
done